code/processes/wdb.q

```diff
@@ -527,20 +524,21 @@ fixpartition:{[subto]
 
 /- for writedown modes partbyenum/default we make sure that partition 0/currentpartition has all the tables.
 /- In that case we can use .Q.chk later to fill the db making it useable for intraday processes
-initmissingtables:{[]
-    .lg.o[`fixpartition;"Adding missing tables(empty) to partition ",string currentpartition];
-    inittable each tablelist[];
-    filldb[];
+initmissingtables:{[currentPt]
```

Comment:
we don't use camel case in TorQ

```suggestion
initmissingtables:{[currentpt]
```

also can you add a comment about what this argument is for? not immediately clear